js/larger.js

```diff
@@ -0,0 +1,14 @@
+// Write a program that reads two Number variables and prints which one is greater. (e.g. "The larger number of 7 and 3 is 7")
+
+// **Requirements**
+// * Your program should maintain two variables
+// * It should correctly identify which is larger and print that result
+
+// CODE BELOW:
+
+let largerNumber = 5
+let smallerNumber = 3
+
+if (largerNumber > smallerNumber) {
+    console.log(`The larger number of ${largerNumber} and ${smallerNumber} is ${largerNumber}`)
+}
```

Comment:
else ( what happens when the 'largerNumber' is actually smaller than the 'smallerNumber' ?)